templates/common/_base/files/configure-ovs-network.yaml

```diff
@@ -433,6 +446,16 @@ contents:
           fi
         fi
       }
+
+      # on every boot we rollback and generate the configuration again, to take
+      # in any changes that have possibly been applied in the standard
+      # configuration sources
+      [ ! -e /run/configure-ovs-boot-done ] && {
```

Comment:
what'st he point of this? To prevent someone from re-running the script causing a re-config?

Reply:
Yes. I don't want to do this if someone runs `systemctl restart ovs-configuration` or the like.